ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1022,7 +1038,7 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
             <OldCdrVersionModal
                 onCancel={() => {
                   this.setState(fp.set(['workspace', 'cdrVersionId'],
-                    getDefaultCdrVersionForTier(this.state.workspace, cdrVersionTiersResponse).cdrVersionId));
+                    getDefaultCdrVersionForTier(this.state.workspace.accessTierShortName, cdrVersionTiersResponse).cdrVersionId));
```

Comment:
updated the signature for getDefaultCdrVersionForTier() because it's sometimes needed without a full Workspace object